test/test_session.py

```diff
@@ -274,6 +276,8 @@ def test_end_sessions(self):
         client = rs_or_single_client(event_listeners=[listener])
         # Start many sessions.
         sessions = [client.start_session() for _ in range(_MAX_END_SESSIONS + 1)]
+        for s in sessions:
+            client.admin.command("ping", session=s)
```

Comment:
This is gonna make the test take a long time. Can we instead just do `s.session_id`?